src/test/run-pass/termination-trait-for-result-box-error_ok.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
```

Comment:
Also, pre-existing, but I would like it if we moved these tests (and the existing ones) into a directory, maybe something like:

```
src/test/$SUITE/rfc1937-termination-trait
```

Reply:
yes; can do

Reply:
`termination_trait` tests reorganized as per above.